src/views/blogPage2.vue

```diff
@@ -0,0 +1,214 @@
+<script setup>
+import { calcularIdade,} from './functions';
+
+</script>
+
+<template>
+  <div>
+    <h2>
+      <span><theButton>Previous</theButton></span
+      >Catalogo de Motos <span><theButton @click="calcularIdade('11/09/1994')">Next</theButton></span>
+    </h2>
+    <ul>
+      <li>
```

Comment:
use um v-for pra fazer isso, transforma os valores que vc precisa em uma variavel e utilize um v-for para printar quantas vezes quiser ao invéz de colocar o mesmo codigo N vezes.